doc/api/fs.markdown

```diff
@@ -1157,6 +1157,17 @@ reliably or at all.
 You can still use `fs.watchFile`, which uses stat polling, but it is slower and
 less reliable.
 
+#### Inodes
+
+<!--type=misc-->
+
+On Linux and OS X systems, `fs.watch` resolves the watched path to an inode. If
+the watched path is deleted and then created so that it has a new inode, you
```

Comment:
@thefourtheye It's consistent with the context, see the previous section on Availability a few lines up:

`You can still use...`